app.js

```diff
@@ -0,0 +1,209 @@
+    sectionHTML.appendChild(h1HTML);
+    var ulHTML = document.createElement('ul');
+    for (var i = 0; i < hours.length; i++) {
+      var li = document.createElement('li');
+      li.innerText = hours[i] + ': ' + this.calculator(i) + ' cookies';
+      ulHTML.appendChild(li);
+    }
+    var totalSales = 0;
+    for (i = 0; i < this.sales.length; i++) {
+      totalSales += this.sales[i];
+    }
+    li.innerText = 'Total Sales: ' + totalSales + ' cookies';
+    ulHTML.appendChild(li);
+    sectionHTML.appendChild(ulHTML);
+  }
+};
+
+var capitolHill = {
+  // MIN CUST= 20
+  //MAX CUST= 38
+  //AVG COOKIE= 2.3
+  name: 'Capitol Hill',
+  minCust: 20,
+  maxCust: 38,
+  avgCookies: 2.3,
+  sales: [],
+  calculator: function(i){
+    this.sales[i]= Math.floor((Math.random() * (this.maxCust - this.minCust) + this.minCust) * this.avgCookies);
+    return this.sales[i];
+  },
+  renderHTML: function() {
+    var sectionHTML = document.createElement('section');
+    document.body.appendChild(sectionHTML);
+    var h1HTML = document.createElement('h1');
+    h1HTML.innerText = this.name;
+    sectionHTML.appendChild(h1HTML);
+    var ulHTML = document.createElement('ul');
+    for (var i = 0; i < hours.length; i++) {
+      var li = document.createElement('li');
+      li.innerText = hours[i] + ': ' + this.calculator(i) + ' cookies';
+      ulHTML.appendChild(li);
+    }
+    var totalSales = 0;
+    for (i = 0; i < this.sales.length; i++) {
+      totalSales += this.sales[i];
+    }
+    li.innerText = 'Total Sales: ' + totalSales + ' cookies';
+    ulHTML.appendChild(li);
+    sectionHTML.appendChild(ulHTML);
+  }
+};
+var alki = {
+  // MIN CUST= 2
+  //MAX CUST= 16
+  //AVG COOKIE= 4.6
+  name: 'alki',
+  minCust: 2,
+  maxCust: 16,
+  avgCookies: 4.6,
+  sales: [],
+  calculator: function(i){
+    this.sales[i]= Math.floor((Math.random() * (this.maxCust - this.minCust) + this.minCust) * this.avgCookies);
+    return this.sales[i];
+  },
+  renderHTML: function() {
+    var sectionHTML = document.createElement('section');
+    document.body.appendChild(sectionHTML);
+    var h1HTML = document.createElement('h1');
+    h1HTML.innerText = this.name;
+    sectionHTML.appendChild(h1HTML);
+    var ulHTML = document.createElement('ul');
+    for (var i = 0; i < hours.length; i++) {
+      var li = document.createElement('li');
+      li.innerText = hours[i] + ': ' + this.calculator(i) + ' cookies';
+      ulHTML.appendChild(li);
+    }
+    var totalSales = 0;
+    for (i = 0; i < this.sales.length; i++) {
+      totalSales += this.sales[i];
+    }
+    li.innerText = 'Total Sales: ' + totalSales + ' cookies';
+    ulHTML.appendChild(li);
+    sectionHTML.appendChild(ulHTML);
+  }
+};
+firstAndPike.calculator();
+firstAndPike.renderHTML();
+seaTacAirport.calculator();
+seaTacAirport.renderHTML();
+seattleCenter.calculator();
+seattleCenter.renderHTML();
+capitolHill.calculator();
+capitolHill.renderHTML();
+alki.calculator();
+alki.renderHTML();
```

Comment:
If you're always doing these two methods back to back, why not create a method on your object literal that just runs those two methods in sequence. Then, in the end down here, just run that one method for each store.